start/client/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I don't think we need to target `es5` since this tutorial is targeted at developers with modern browsers. Is this the generic `tsconfig.json` generated by CRA?

Reply:
@peggyrayzis Yeah, CRA automatically adds the `tsconfig.json` with these settings for us.
https://create-react-app.dev/docs/adding-typescript/